### PR TITLE
Small UI fixes and additions in Auth

### PR DESCRIPTION
**Pull request by SaraVieira**
- Changes layout of start
CleanShot 2021-06-13 at 16 30 13@2x

- Adds logout button
CleanShot 2021-06-13 at 16 30 21@2x

- When you finish it uses your signed-in data to send your score instead of asking for your name

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -32,6 +32,8 @@ export const setupSession = (set) => {
   })
 }
 
+export const signOut = () => client.auth.signOut()
```
Comment (njm222) [inline src/data.js:35]: duplicate code `unAuthenticateUser`

File: src/styles.css [hunk 2]
```diff
@@ -37,19 +37,31 @@ body {
...
-.auth-provider { 
+.auth-provider {
   width: 250px;
   height: auto;
-  padding: 1em;
   margin: 1em 0;
   display: flex;
   justify-content: space-evenly;
   align-items: center;
+  background: transparent;
```
Comment (Gusted) [inline src/styles.css:47]:
```suggestion
  background-color: transparent;
```

File: src/ui/Auth.jsx [hunk 3]
```diff
@@ -1,17 +1,31 @@
 import { authenticateUser } from '../data'
-import googleLogo from '../icons/google-logo.png'
-import githubLogo from '../icons/github-logo.png'
 
 const providers = [
   {
     provider: 'google',
     label: 'Sign in with Google',
-    logo: googleLogo,
+    Logo: () => (
```
Comment (njm222) [inline src/ui/Auth.jsx:7]: https://developers.google.com/identity/branding-guidelines

File: src/ui/Auth.jsx [hunk 4]
```diff
@@ -1,17 +1,31 @@
...
+      <svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+        <title>Google</title>
+        <path
+          fill="#fff"
+          d="M12.48 10.92v3.28h7.84c-.24 1.84-.853 3.187-1.787 4.133-1.147 1.147-2.933 2.4-6.053 2.4-4.827 0-8.6-3.893-8.6-8.72s3.773-8.72 8.6-8.72c2.6 0 4.507 1.027 5.907 2.347l2.307-2.307C18.747 1.44 16.133 0 12.48 0 5.867 0 .307 5.387.307 12s5.56 12 12.173 12c3.573 0 6.267-1.173 8.373-3.36 2.16-2.16 2.84-5.213 2.84-7.667 0-.76-.053-1.467-.173-2.053H12.48z"
```
Comment (Gusted) [inline src/ui/Auth.jsx:12]:
```suggestion
          d="M12.5 11v3.2h7.8a7 7 0 0 1-1.8 4.1 8 8 0 0 1-6 2.4c-4.8 0-8.6-3.9-8.6-8.7a8.6 8.6 0 0 1 14.5-6.4l2.3-2.3C18.7 1.4 16 0 12.5 0 5.9 0 .3 5.4.3 12S6 24 12.5 24a11 11 0 0 0 8.4-3.4c2.1-2.1 2.8-5.2 2.8-7.6 0-.8 0-1.5-.2-2h-11z"
```
It doesn't have to be _that_ precise.

File: src/ui/Auth.jsx [hunk 5]
```diff
@@ -1,17 +1,31 @@
...
   {
     provider: 'github',
     label: 'Sign in with Github',
-    logo: githubLogo,
+    Logo: () => (
+      <svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+        <title>GitHub</title>
+        <path
+          fill="#fff"
+          d="M12 .297c-6.63 0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577 0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93 0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176 0 0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22 0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22 0 1.606-.015 2.896-.015 3.286 0 .315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12"
```
Comment (Gusted) [inline src/ui/Auth.jsx:25]:
```suggestion
          d="M12 .3a12 12 0 0 0-3.8 23.4c.6.1.8-.3.8-.6v-2c-3.3.7-4-1.6-4-1.6-.6-1.4-1.4-1.8-1.4-1.8-1-.7.1-.7.1-.7 1.2 0 1.9 1.2 1.9 1.2 1 1.8 2.8 1.3 3.5 1 0-.8.4-1.3.7-1.6-2.7-.3-5.5-1.3-5.5-6 0-1.2.5-2.3 1.3-3.1-.2-.4-.6-1.6 0-3.2 0 0 1-.3 3.4 1.2a11.5 11.5 0 0 1 6 0c2.3-1.5 3.3-1.2 3.3-1.2.6 1.6.2 2.8 0 3.2a4 4 0 0 1 1.3 3.2c0 4.6-2.8 5.6-5.5 5.9.5.4.9 1 .9 2.2v3.3c0 .3.1.7.8.6A12 12 0 0 0 12 .3"
```
Same story

**Review by njm222 — CHANGES_REQUESTED** [review]

**Comment by njm222** [comment]
Looks much better, thanks!

**Review by Gusted — APPROVED** [review]
Perfection 👌

File: src/ui/Auth.jsx (hunk 3 above)
Comment (SaraVieira) [reply src/ui/Auth.jsx:7]: hate it but fixed
CleanShot 2021-06-13 at 17 37 49@2x

**Comment by SaraVieira** [comment]
@Gusted I had to change the logo for google and design, can you please review again?

**Comment by Gusted** [comment]
> can you please review again?

But of course, madam.

File: src/ui/Auth.jsx [hunk 6]
```diff
@@ -1,17 +1,30 @@
...
   {
     provider: 'google',
     label: 'Sign in with Google',
-    logo: googleLogo,
+    Logo: () => (
+      <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 70 72">
+        <path fill="#4285F4" fillRule="evenodd" d="M70 37l-1-7H36v14h19c-1 4-3 8-7 10v9h11c7-6 11-15 11-26z" clipRule="evenodd" />
+        <path fill="#34A853" fillRule="evenodd" d="M36 72c9 0 17-3 23-9l-11-9a21 21 0 01-32-11H4v9c6 12 18 20 32 20z" clipRule="evenodd" />
+        <path fill="#FBBC05" fillRule="evenodd" d="M16 43a21 21 0 010-13V20H4a36 36 0 000 32l12-9z" clipRule="evenodd" />
+        <path fill="#EA4335" fillRule="evenodd" d="M36 15c5 0 10 2 13 5l11-10c-7-6-15-9-24-9C22 1 10 9 4 20l12 10c3-9 10-15 20-15z" clipRule="evenodd" />
```
Comment (Gusted) [inline src/ui/Auth.jsx:12]:
```suggestion
        <path fill="#4285F4" d="m70 37-1-7H36v14h19c-1 4-3 8-7 10v9h11c7-6 11-15 11-26z"/>
        <path fill="#34A853" d="M36 72c9 0 17-3 23-9l-11-9a21 21 0 0 1-32-11H4v9c6 12 18 20 32 20z"/>
        <path fill="#FBBC05" d="M16 43a21 21 0 0 1 0-13V20H4a36 36 0 0 0 0 32l12-9z"/>
        <path fill="#EA4335" d="M36 15c5 0 10 2 13 5l11-10A36.6 36.6 0 0 0 4 20l12 10c3-9 10-15 20-15z"/>
```
fillRule & clipRule are not use-able in this situation and doesn't effect the output. + Minifcation :D

Comment (SaraVieira) [reply src/ui/Auth.jsx:12]: You seem to know your SVG's! Thank you, did not know that :)

**Pull request merged**